[2020-12-11T12:35:19.421Z]
[2020-12-11T12:35:19.421Z] ----------------------------------------------------------------------
[2020-12-11T12:35:19.421Z] FAIL: lock_test.go:655: testLockSuite.TestBatchResolveTxnFallenBackFromAsyncCommit
[2020-12-11T12:35:19.421Z]
[2020-12-11T12:35:19.421Z] lock_test.go:656:
[2020-12-11T12:35:19.421Z]     s.prepareTxnFallenBackFromAsyncCommit(c)
[2020-12-11T12:35:19.421Z] lock_test.go:612:
[2020-12-11T12:35:19.421Z]     c.Assert(err, IsNil)
[2020-12-11T12:35:19.421Z] ... value *errors.fundamental = commit TS 421446278553534517 is too large ("commit TS 421446278553534517 is too large")
[2020-12-11T12:35:19.421Z]
[2020-12-11T12:35:19.421Z] [2020/12/11 20:32:46.463 +08:00] [INFO] [client.go:193] ["[pd] create pd client with endpoints"] [pd-address="[127.0.0.1:2379]"]
[2020-12-11T12:35:19.421Z] [2020/12/11 20:32:46.465 +08:00] [INFO] [base_client.go:308] ["[pd] switch leader"] [new-leader=http://127.0.0.1:2379] [old-leader=]
[2020-12-11T12:35:19.421Z] [2020/12/11 20:32:46.465 +08:00] [INFO] [base_client.go:112] ["[pd] init cluster id"] [cluster-id=6904975592488621718]
[2020-12-11T12:35:19.421Z] [2020/12/11 20:32:46.470 +08:00] [INFO] [lock_resolver.go:773] ["resolve async commit"] [startTS=421446278553534516] [commitTS=0]
[2020-12-11T12:35:19.422Z] [2020/12/11 20:32:46.473 +08:00] [WARN] [snapshot.go:589] ["2PC failed"] [error="commit TS 421446278566641673 is too large"] [errorVerbose="commit TS 421446278566641673 is too